apps/parse-rated-paragraphs/src/app/parse-rated.ts

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs';
```

Comment:
looks like this file was changed in a different commit right?

Reply:
Oh this needs to be merged back into lev/rate-articles